src/output-json.c

```diff
@@ -117,6 +118,28 @@ void SCJsonDecref(json_t *json)
     json_decref(json);
 }
 
+json_t *SCJsonString(const char *val)
+{
+    json_t * retval = json_string(val);
+    char retbuf[MAX_JSON_SIZE] = {0};
+    if (retval == NULL) {
+        uint32_t u = 0;
+        uint32_t offset = 0;
+        for (u = 0; u < strlen(val); u++) {
```

Comment:
Getting lots of failed tests, crashing here:
<pre>
ASAN:SIGSEGV
=================================================================
==60469==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000000 (pc 0x7fd0ba5d7746 bp 0x7fd0b4978c60 sp 0x7fd0b49783e8 T2)
    #0 0x7fd0ba5d7745 in strlen (/lib/x86_64-linux-gnu/libc.so.6+0x8b745)
    #1 0x7fd0bd5a61a5 in __interceptor_strlen (/usr/lib/x86_64-linux-gnu/libasan.so.2+0x701a5)
    #2 0xefef84 in SCJsonString /home/victor/devel/eidps/src/output-json.c:128
    #3 0xedbd8c in JsonTlsLogFingerprint /home/victor/devel/eidps/src/output-json-tls.c:142
    #4 0xedd1af in JsonTlsLogJSONExtended /home/victor/devel/eidps/src/output-json-tls.c:360
    #5 0xedd5e1 in JsonTlsLogger /home/victor/devel/eidps/src/output-json-tls.c:420
    #6 0xefc901 in OutputTxLog /home/victor/devel/eidps/src/output-tx.c:236
    #7 0xe7408a in OutputLoggerLog /home/victor/devel/eidps/src/output.c:920
    #8 0xdf9fcd in FlowWorker /home/victor/devel/eidps/src/flow-worker.c:263
    #9 0x113e20a in TmThreadsSlotVarRun /home/victor/devel/eidps/src/tm-threads.c:143
    #10 0x1141649 in TmThreadsSlotVar /home/victor/devel/eidps/src/tm-threads.c:598
    #11 0x7fd0bb98e6b9 in start_thread (/lib/x86_64-linux-gnu/libpthread.so.0+0x76b9)
    #12 0x7fd0ba65341c in clone (/lib/x86_64-linux-gnu/libc.so.6+0x10741c)
</pre>

Reply:
Oh, null value passed to SCJsonString. I guess that's why most of the tests are failing. I will insert a check for it.